src/app/shared/constants/flags/inhabit-type.ts

```diff
@@ -3,6 +3,6 @@ import { Flag } from '../../types/general';
 export const INHABIT_TYPE: Flag[] = [
   { bit: 0, name: 'GROUND' },
   { bit: 1, name: 'WATER' },
-  { bit: 2, name: 'AIR' },
+  { bit: 2, name: 'FLYING' },
```

Comment:
are you sure it's not the wiki that should be updated? `AIR` is consistent with `GROUND` and `WATER`.... while `FLYING` is like `WALKING` and `SWIMMING` ...

Reply:
Yea.. core also has air